scenes/globals/enums.gd

```diff
@@ -0,0 +1,9 @@
+# SPDX-FileCopyrightText: The Threadbare Authors
+# SPDX-License-Identifier: MPL-2.0
+class_name Enums
```

Comment:
I think that the suggestion was to come up with the following class hierarchy:
- Character
  - Player
  - NPC
 
And this enum can go to Character.

Reply:
Signs also use this enum!

Reply:
Ah OK, the commit didn't say anything about the signs but I see now. In any case the name `enums.gd` or having a global named `Enums` is too broad, in my opinion. If it lives in a class named `Character` then signs can reference eg. `Character.LookAtSide.LEFT`. Or if a global is preferred, could be `Directions` a better name?

Reply:
But then we would do `Directions.LookAtSide.LEFT`?

There's a limitation in Godot where we cannot have named enums that are available globally. You can have an unnamed enum in a class and access those values as constants, but you cannot use it as a type. So we need the enum to be named and to be inside a class. My thinking was that we could have `Enums` be like a namespace, of all the "global" enums.

Reply:
I don't love these kinds of "misc" namespaces but I don't really see a better home for this enum, given the limitations you point out. OK by me!